src/VirtualMonoRepo/eng/common/templates/jobs/vmr-synchronization.yml

```diff
@@ -0,0 +1,76 @@
+###
+### DO NOT MODIFY THIS FILE!
+###
+### This YAML was auto-generated from VmrSyncJobTemplate
+### To make changes, change the C# definition and rebuild its project
+###
+### These steps synchronize code from product repositories into the VMR (https://github.com/dotnet/dotnet)
+
+parameters:
+- name: targetRef
+  displayName: Target revision in dotnet/installer to synchronize
+  type: string
+  default: $(Build.SourceVersion)
+
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to use
+  type: string
+  default: $(Build.SourceBranchName)
+
+jobs:
+- job: Synchronize
+  displayName: Synchronize dotnet/dotnet
+  pool:
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      name: NetCore-Public
+      demands:
+      - ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
```

Comment:
Can we use a newer Ubuntu version? Ubuntu 18.04 is going eol in a few months (April 2023)

Reply:
Yeah, makes sense. I just copied what other pipelines here use but this build doesn't care about the specifics so makes sense to just use 2004